.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "gutenberg"]
 	path = gutenberg
-	url = ../../WordPress/gutenberg.git
+	url = https://github.com/derekblank/gutenberg.git
```

Comment:
The URL of the Gutenberg submodule should remain pointing to its original value `../../WordPress/gutenberg.git`.